File: ml-a-z/Simple_Linear_Regression/milam_regression.py
# Importing the libraries
import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

# Importing the dataset
dataset = pd.read_csv('Salary_Data.csv')
dataset

X = dataset.iloc[:, :-1].values
y = dataset.iloc[:, 1].values
X, y

# Splitting the dataset into the Training set and Test set
from sklearn.cross_validation import train_test_split
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size = 0.33, random_state = 0)
X_train
X_test
y_train
y_test

File: ml-a-z/Simple_Linear_Regression/milam_regression.R
# Data Preprocessing Template

# Importing the dataset
dataset = read.csv('Salary_Data.csv')

# Splitting the dataset into the Training set and Test set
# install.packages('caTools')
library(caTools)
set.seed(123)
split = sample.split(dataset$Salary, SplitRatio = 0.67)
training_set = subset(dataset, split == TRUE)
test_set = subset(dataset, split == FALSE)

# fit simple linear regression to trainging set
regr = lm(formula = Salary ~ YearsExperience, 
          data = training_set)

